### PR TITLE
Net5.0 cleanup

### PR DESCRIPTION
**Pull request by cijothomas**
Fixes #.

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: test/OpenTelemetry.Instrumentation.AspNetCore.Tests/OpenTelemetry.Instrumentation.AspNetCore.Tests.csproj [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net5.0'">
     <ProjectReference Include="$(RepoRoot)\test\TestApp.AspNetCore.5.0\TestApp.AspNetCore.5.0.csproj" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="5.0.0-rc.2.20475.17" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="5.0.0" />
```
Comment (reyang) [inline test/OpenTelemetry.Instrumentation.AspNetCore.Tests/OpenTelemetry.Instrumentation.AspNetCore.Tests.csproj:27]: Not blocking this PR, we might want to extract these versions as well (e.g. `MicrosoftAspNetCoreMvcTestingNet5PkgVer`).

**Review by reyang — APPROVED** [review]
LGTM.

**Comment by eddynaka** [comment]
@cijothomas , we have a few other places: https://github.com/open-telemetry/opentelemetry-dotnet/search?l=XML&q=rc&type=code

**Comment by eddynaka** [comment]
> @cijothomas , we have a few other places: https://github.com/open-telemetry/opentelemetry-dotnet/search?l=XML&q=rc&type=code

Thanks! :)

**Pull request merged**